script/testing/junit/traces/select.test

```diff
@@ -150,3 +150,336 @@ SELECT * FROM temp3 WHERE id=1 OR time=300 AND id=3 OR id=4 AND time=200 OR id=2
 
 statement ok
 DROP TABLE temp3
+
+# LIKE AND NOT LIKE
+statement ok
+CREATE TABLE like_tbl (int_val INT, double_val DECIMAL,str_i_val VARCHAR(32),str_a_val VARCHAR(32),is_null INT);
+
+statement ok
+INSERT INTO like_tbl (int_val, double_val, str_i_val, str_a_val, is_null) VALUES (1, 2.0, '123456', 'AbCdEf', 0);
```

Comment:
Sorry, but can you update the test to include more than one string? I am concerned about this case happening in the future:

- we break LIKE and it matches any string
- but it is not caught because there's only one value with is_null = 0 anyway

Suggestions:

```
INSERT INTO like_tbl (int_val, double_val, str_i_val, str_a_val, is_null) VALUES (1, 2.0, '123456', 'AbCdEF', 0);
INSERT INTO like_tbl (int_val, double_val, str_i_val, str_a_val, is_null) VALUES (1, 2.0, '123456', 'AbcdEf', 0);
INSERT INTO like_tbl (int_val, double_val, str_i_val, str_a_val, is_null) VALUES (1, 2.0, '123456', 'abcdEf', 0);
```

Reply:
Also, I'm not sure why there is a is_null column to begin with. Is it possible to do WHERE str_a_val IS NOT NULL?

Reply:
```
terrier=# create table foo (a text);
CREATE TABLE
terrier=# insert into foo values (null);
INSERT 0 1
terrier=# insert into foo values ('a');
INSERT 0 1
terrier=# select * from foo where a is not null;
 a 
---
 a
(1 row)
```

Seems to work for me, and is less hacky looking.

Reply:
I think we should put some of the `LIKE` in the `WHERE` clause just to be safe.

Reply:
added